pandas/tests/arrays/sparse/test_array.py

```diff
@@ -9,6 +9,8 @@
 from pandas.compat import range
 import pandas.util._test_decorators as td
 
+from pandas.core.dtypes.missing import isna
```

Comment:
in tests just
``from pandas import isna``

Reply:
changed